core/src/components/datetime/datetime.tsx

```diff
@@ -1893,6 +1893,7 @@ export class Datetime implements ComponentInterface {
                   aria-hidden="true"
                   icon={this.showMonthAndYear ? expandedIcon : collapsedIcon}
                   lazy={false}
+                  flipRtl
```

Comment:
```suggestion
                  flipRtl={true}
```
`flipRtl` accepts a boolean https://github.com/ionic-team/ionicons/blob/main/src/components/icon/icon.tsx#L47 (though what you have right now does work)